src/current/v25.3/citext.md

```diff
@@ -0,0 +1,126 @@
+---
+title: CITEXT
+summary: The CITEXT data type stores case-insensitive text values.
+toc: true
+docs_area: reference.sql
+---
+
+The `CITEXT` [data type]({% link {{ page.version.version }}/data-types.md %}) stores case-insensitive strings.
+
+All `CITEXT` values are folded to lowercase before comparison. This is handled internally with the [`lower()`]({% link {{ page.version.version }}/functions-and-operators.md %}#string-and-byte-functions) function.
+
+For example, the following comparison evaluates to `true`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT 'Roach'::CITEXT = 'roach'::CITEXT;
+~~~
+
+~~~
+  ?column?
+------------
+     t
+~~~
+
+With `CITEXT`, equality operators (`=`, `!=`, `<>`), ordering operators (`<`, `>`, etc.), and [`STRING` functions]({% link {{ page.version.version }}/functions-and-operators.md %}#string-and-byte-functions), treat values as case-insensitive by default. Refer to the [example](#example).
```

Comment:
Is there an example of one of these string functions that treat the values as case-insensitive? I think this might not be true.

Reply:
Actually nm. Although Paul confirmed the statement is true, I don't know if there is a good example of a function that demonstrates the case insensitivity. So I'm removing this line for simplicity.